README.md

```diff
@@ -1,42 +1,160 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-#page
+<div align="center">
 
+  <img src="./Images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-## Built With
+</div>
 
-- HTML
-- CSS
-- JavaScript
-## Link
-https://eddyshimwa.github.io/Portforio/
+# 📗 Table of Contents
 
-## Link
-https://eddyshimwa.github.io/Portforio/
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-👤 **Author**
+# 📖 [Creative Commons Global Summit] <a name="about-project"></a>
+
+> 
+
+**[Creative Commons Global Summit]** Is a first capstone project in microverse, It includes all the lessons we learned in 4 weeks including HTML, javaScript, CSS, bootstrap 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Completed Mobile version]**
+- **[Completed Desktop version]**
+- **[Added functionalities on menu buttons for mobile version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://eddyshimwa.github.io/CapStone-Project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+### Usage
+
+
+
+### Run tests
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors 
+
+👤 **Author1**
 
 - GitHub: [@EddyShimwa](https://github.com/EddyShimwa)
 - Twitter: [@eddy56388406](https://twitter.com/eddy56388406)
-- LinkedIn: [Pray Eddy SHIMWA]([https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/]
+- LinkedIn: [Pray Eddy SHIMWA]([https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/])
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
 
+> Describe 1 - 3 features you will add to the project.
 
-## 🤝 Contributing
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```

Comment:
- Hey, you should some features about your webpage here to give users an idea of what your webpage does, or you can choose to remove it if you don't want to add some. Leaving it empty doesn't make you more of a professional.

Reply:
Yeah Sure! Thanks!